app/helpers/audits_helper.rb

```diff
@@ -13,7 +13,7 @@ def id_to_label(name, change)
       when /.*_id$/
         name.classify.gsub('Id','').constantize.find(change).to_label
       else
-        change.to_s
+        change.to_s.include?(EncryptValue::ENCRYPTION_PREFIX) ? _("[encrypted]") : change.to_s
```

Comment:
Thanks, that's better, but there aren't any tests for the functionality - please add to test/models/concerns/audit_extensions_test.rb.